strategies.nls

```diff
@@ -165,3 +153,50 @@ end
 
 
 
+
+
+; procedure that lets the agents act on the just computed best theory
```

Comment:
this procedure would also fit into "behavior.nls"
I still wonder if it would not make more sense to have one module "evaluation.nls" and another "acting.nls" and put this into acting.nls. Then the conceptual distinction would seem more fine-grained. 
I think for this merge we should keep things as you did, but maybe we could think about this in future.